GNUmakefile

```diff
@@ -35,3 +35,8 @@ generate: install-packer-sdc
 	packer-sdc renderdocs -src ./docs -dst ./.docs -partials ./docs-partials
 	# checkout the .docs folder for a preview of the docs
 
+build-docs: install-packer-sdc
```

Comment:
Why do we still leave the older generate task in place, should we only use build-docs after this PR is merged?

Reply:
If you're referring to the release pipeline. We are keeping everything the same until we've cut over the plugins to production. Once live we can remove the doc.zip generation. But it should not affect anything if left in place.